source/core/rtc_adapter/VideoSendAdapter.cc

```diff
@@ -179,6 +179,12 @@ bool VideoSendAdapterImpl::init()
     configuration.event_log = m_eventLog.get();
     configuration.retransmission_rate_limiter = m_retransmissionRateLimiter.get();
     configuration.local_media_ssrc = m_ssrc; //rtp_config.ssrcs[i];
+    // TODO: enable UlpfecGenerator
```

Comment:
should follow similar logic as webrtc/call/rtp_video_sender.cc:MaybeCreateFecGenerator() for creating the fec genetator, instead of hardcoding it to ulpfec_generator.